front/lib/swr/mcp_servers.ts

```diff
@@ -436,38 +444,31 @@ export function useSyncRemoteMCPServer(
 }
 
 /**
- * Hook to update an MCP server
+ * Hook to update a MCP server
```

Comment:
nit, it was correct with "an" 😄

```suggestion
 * Hook to update an MCP server
```

Reply:
thanks